include/swift/AST/Identifier.h

```diff
@@ -276,13 +276,26 @@ class DeclName {
   /// Retrieve the 'base' name, i.e., the name that follows the introducer,
   /// such as the 'foo' in 'func foo(x:Int, y:Int)' or the 'bar' in
   /// 'var bar: Int'.
-  Identifier getBaseName() const {
+  Identifier getIdentifier() const {
```

Comment:
This isn't really correct. You should only be able to call `getIdentifier` on a simple DeclName, since compound ones have multiple identifiers. And then I'd consider naming it something different too.